appveyor-windows.yml

```diff
@@ -63,6 +63,36 @@ install:
   # Install AWS CLI Globally via pip3
   - "pip install awscli"
 
+  # Switch to Docker Linux containers
+  - ps: Switch-DockerLinux
+
+  # Check for git executable
+  - "git --version"
+
+  # Get testing env vars
+  - ps: "python -m virtualenv venv_env_vars"
+  - ps: "./venv_env_vars/Scripts/pip install boto3"
+  - ps: "$test_env_var = ./venv_env_vars/Scripts/python tests/get_testing_resources.py"
+  - ps: "$test_env_var_json = ConvertFrom-Json $test_env_var"
+
+  - ps: "$env:CI_ACCESS_ROLE_AWS_ACCESS_KEY_ID = $env:AWS_ACCESS_KEY_ID"
+  - ps: "$env:CI_ACCESS_ROLE_AWS_SECRET_ACCESS_KEY = $env:AWS_SECRET_ACCESS_KEY"
+  - ps: "$env:CI_ACCESS_ROLE_AWS_SESSION_TOKEN = $env:AWS_SESSION_TOKEN"
+
+  - ps: "aws sts get-caller-identity"
+
+  - ps: "$env:AWS_ACCESS_KEY_ID = $test_env_var_json.accessKeyID"
+  - ps: "$env:AWS_SECRET_ACCESS_KEY = $test_env_var_json.secretAccessKey"
+  - ps: "$env:AWS_SESSION_TOKEN = $test_env_var_json.sessionToken"
+  - ps: "$env:TASK_TOKEN = $test_env_var_json.taskToken"
+  - ps: "$env:AWS_S3_TESTING = $test_env_var_json.TestBucketName"
+  - ps: "$env:AWS_ECR_TESTING = $test_env_var_json.TestECRURI"
+  - ps: "$env:AWS_KMS_KEY = $test_env_var_json.TestKMSKeyArn"
+  - ps: "$env:AWS_SIGNING_PROFILE_NAME = $test_env_var_json.TestSigningProfileName"
+  - ps: "$env:AWS_SIGNING_PROFILE_VERSION_ARN = $test_env_var_json.TestSigningProfileARN"
+
+  - ps: "aws sts get-caller-identity"
```

Comment:
Should we move this part into python file so that we can call it from both windows and linux jobs?

Reply:
We can't set env vars in a parent process from a child process. This must be done inside of the CI script.